### PR TITLE
DO NOT MERGE

### PR DESCRIPTION
**Pull request by vigsterkr**
this is a CI debug pr

### CODE REVIEW

File: src/shogun/kernel/Kernel.h [hunk 1]
```diff
@@ -27,7 +27,6 @@ namespace shogun
 {
 	class CFile;
 	class CFeatures;
-	class CKernelNormalizer;
```
Comment (gf712) [inline src/shogun/kernel/Kernel.h]: @vigsterkr Could this be an issue?

Comment (vigsterkr) [reply src/shogun/kernel/Kernel.h]: unlikely as we have `#include <shogun/kernel/normalizer/KernelNormalizer.h>` before the class :)

Comment (gf712) [reply src/shogun/kernel/Kernel.h]: OK! just finding all the kernelnormalizer that were changed...

**Comment by vigsterkr** [comment]
@lisitsyn @karlnapf finally this fix is soon to be finished (the integration tester seems to be having some issue as well some unit tests...) all in all plz take a look at this pr, and lemme know if you have any objections to merge it as soon as the tests are passing.

**Comment by karlnapf** [comment]
What does this do/fix?
So now all classes need this export thing?

**Comment by vigsterkr** [comment]
windows build... as currently it was barely working.... and yes we need export for all the classes you intend to expose to the outside world....

**Comment by karlnapf** [comment]
well that's great then!
I am not super keen on having this export statement there, but hey if it makes stuff work why not :)

**Comment by vigsterkr** [comment]
closing in favour of #4719

**Pull request closed without merging**